File: majorProject/src/test/java/com/aocc/majorproject/display/SecondaryDisplayFinderTest.java
package com.aocc.majorproject.display;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class SecondaryDisplayFinderTest {

    @Test
    public void categoryRear_matchesFrameworkCategory() {
        assertEquals("android.hardware.display.category.REAR",
                SecondaryDisplayFinder.CATEGORY_REAR);
    }
}

File: majorProject/src/test/kotlin/com/aocc/majorproject/display/SecondaryDisplayFinderTest.kt
package com.aocc.majorproject.display

import org.junit.Assert.assertEquals
import org.junit.Test

class SecondaryDisplayFinderTest {

    @Test
    fun categoryRear_matchesFrameworkCategory() {
        assertEquals(
            "android.hardware.display.category.REAR",
            SecondaryDisplayFinder.CATEGORY_REAR
        )
    }
}
